bin/index.js

```diff
@@ -65,3 +63,5 @@ Please fix the error and try again.
 )
     process.exitCode = 1
   })
+  // remove the blow when https://blinkmobile.atlassian.net/browse/CC-22 is done
+  .then(() => process.listenerCount('SIGINT') > 0 ? process.exit() : true)
```

Comment:
yeah, if `process.exit` is called without a parameter the last value of `process.exitCode` is used:

https://nodejs.org/api/process.html#process_process_exitcode